Assets/Scripts/CountdownController.cs

```diff
@@ -5,9 +5,21 @@
 
 public class CountdownController : MonoBehaviour
 {
-    private static float countdownTime = 3f;
+    public static CountdownController Instance { get; private set; }
+    public float countdownTime = 3f;
```

Comment:
this field should be private. To allow us to modify it simply add the SerializeField attribute to it and it'll appear in the unity editor